src/commons/util.js

```diff
@@ -525,8 +525,8 @@ export function* domains(urlOrHost) {
 
 export function tld(urlstr, upto = 2, d = ".") {
   if (emptyString(urlstr)) return "";
-  // convert a domain-name of form x.y.tld to url http://x.y.tld
-  if (!urlstr.includes("://")) urlstr = "http://" + urlstr;
+  // convert a domain-name of form x.y.tld to url https://x.y.tld
+  if (!urlstr.includes("://")) urlstr = "https://" + urlstr;
```

Comment:
this method abuses the `URL` class to extract the `hostname`; and so, the protocol used doesn't really make much of a material difference.